src/main/java/com/radixdlt/atommodel/tokens/TokensConstraintScrypt.java

```diff
@@ -492,6 +492,23 @@ private void defineStaking(SysCalls os) {
 			),
 			(in, meta) -> checkSignedBy(meta, in.getAddress())
 		));
+
+		// Stake movement
+		os.executeRoutine(new CreateFungibleTransitionRoutine<>(
```

Comment:
I might be wrong, but I believe this allows someone to just move StakedTokens around to other users. Perhaps there should be a check that it's being sent to the same address.

Reply:
OK, will add.